stock_analysis/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# © 2016 Lorenzo Battistini - Agile Business Group - I.A.S. Ingenieria,
+# Aplicaciones y Software
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Analysis",
+    "summary": "Analysis view for stock",
+    "version": "9.0.0.1.1",
+    "category": "Inventory, Logistic, Storage",
+    "website": "https://www.agilebg.com - http://www.ias.com.co/",
+    "author": "Agile Business Group, Odoo Community Association (OCA), \
```

Comment:
Use this notation and you should short your name as the author name is limited to 128 chars:
```
    "author": "Agile Business Group,"
              "Odoo Community Association (OCA),"
              "I.A.S."
```